rvps/src/store/local_fs/mod.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! This Store stores RV information inside a local file
+
+use std::path::Path;
+
+use anyhow::*;
+
+use crate::{ReferenceValue, Store};
+
+/// Local directory path to store the reference values,
+/// which is created by sled engine.
+const FILE_PATH: &str = "./reference_values";
```

Comment:
Will this file be inside the `ATTESTATION_SERVER_WORKDIR` ?

Reply:
Yes, agreed with you. Now the new PR of AS doesn't have a `ATTESTATION_SERVER_WORKDIR` const, so I suppose to use the previous `/opt/attestation-server` in RVPS